drydock/templates/kustomized/global-stack/kustomization.yml

```diff
@@ -0,0 +1,28 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: {{ DRYDOCK_GLOBAL_STACK_NAMESPACE }}
+
+{%- if DRYDOCK_GLOBAL_RESOURCES|default(false) %}
+resources:
+  {{ patch("drydock-global-resources") }}
+{%- endif %}
+
+helmCharts:
+- name: kube-prometheus-stack
+  repo: https://prometheus-community.github.io/helm-charts
+  version: {{ DRYDOCK_PROMETHEUS_VERSION }}
+  releaseName: prometheus
+  includeCRDs: true
+{%- if DRYDOCK_PROMETHEUS_VALUES | default(false) %}
+  valuesInline:
+    {{ DRYDOCK_PROMETHEUS_VALUES | indent(4) }}
+{%- endif %}
+{{ patch("drydock-global-helm-charts") }}
+
+{%- if DRYDOCK_GLOBAL_OVERLAYS | default(false) %}
+patches:
+{%- if DRYDOCK_PROMETHEUS_OVERLAYS | default(false) %}
+  {{ DRYDOCK_PROMETHEUS_OVERLAYS }}
```

Comment:
We should use `_PATCHES` instead of  `_OVERLAYS`, we usually refer to the values in the `resources:` section as overlays.

I think we can go without `DRYDOCK_GLOBAL_OVERLAYS`. Instead of having an additional variable to enable or disable all the patches, we can simply chose to not define the `DRYDOCK_PROMETHEUS_OVERLAYS` and have a `patches:` section empty.